### PR TITLE
Adding Minimal Label:Project Page #428

### PR DESCRIPTION
**Pull request by Chaitanya31612**
Modified the tag on the project card to a minimal and modern one. Chosen this color as it looked aesthetic and maintained the flow of the site as the image above also has such color.

image

image
image

#### (SWOC)

### CODE REVIEW

**Review by anushbhatia — COMMENTED** [review]
Can add a logo related to the tech stack will make it look more beautiful. If possible

**Comment by Chaitanya31612** [comment]
image
image
image

**Comment by Chaitanya31612** [comment]
@anushbhatia please review it, I've added the icons for every tags.

**Review by smaranjitghose — APPROVED** [review]

**Comment by smaranjitghose** [comment]
Nice work @Chaitanya31612!

How do you propose we have multiple tags for the same card? Including the separate tag images themselves.

@anushbhatia your further opinions?


# Additionally:

- Create a tag.MD file on the root level of the repo.
- Give the links to the most common tags including a side demo (similar to what we do for GitHub badges)
React, HTML, CSS, JS, Python, TensorFlow, Keras, PyTorch, PyCaret, Scikit-Learn, opencv, socket.io
- Give a standalone PR for this, once this PR is merged

**Comment by Chaitanya31612** [comment]
> Nice work @Chaitanya31612!
> 
> How do you propose we have multiple tags for the same card? Including the separate tag images themselves.
> 
> @anushbhatia your further opinions?
> 
> # Additionally:
> * Create a tag.MD file on the root level of the repo.
> * Give the links to the most common tags including a side demo (similar to what we do for GitHub badges)
>   React, HTML, CSS, JS, Python, TensorFlow, Keras, PyTorch, PyCaret, Scikit-Learn, opencv, socket.io
> * Give a standalone PR for this, once this PR is merged

Thanks, @smaranjitghose 😄 
Multiple tags sound good, working on it.

Can you elaborate on what links and side demo are you talking about in the tag.md file?

**Comment by Chaitanya31612** [comment]
> Nice work @Chaitanya31612!
> 
> How do you propose we have multiple tags for the same card? Including the separate tag images themselves.
> 
> @anushbhatia your further opinions?
> 
> # Additionally:
> * Create a tag.MD file on the root level of the repo.
> * Give the links to the most common tags including a side demo (similar to what we do for GitHub badges)
>   React, HTML, CSS, JS, Python, TensorFlow, Keras, PyTorch, PyCaret, Scikit-Learn, opencv, socket.io
> * Give a standalone PR for this, once this PR is merged

image

image

@smaranjitghose how about this?

**Comment by Chaitanya31612** [comment]
> > Nice work @Chaitanya31612!
> > How do you propose we have multiple tags for the same card? Including the separate tag images themselves.
> > @anushbhatia your further opinions?
> > # Additionally:
> > 
> > * Create a tag.MD file on the root level of the repo.
> > * Give the links to the most common tags including a side demo (similar to what we do for GitHub badges)
> >   React, HTML, CSS, JS, Python, TensorFlow, Keras, PyTorch, PyCaret, Scikit-Learn, opencv, socket.io
> > * Give a standalone PR for this, once this PR is merged
> 
> image
> 
> image
> 
> @smaranjitghose how about this?

@smaranjitghose can you please review this pull request and please explain what side demo are you talking about?

**Comment by smaranjitghose** [comment]
> > Nice work @Chaitanya31612!
> > How do you propose we have multiple tags for the same card? Including the separate tag images themselves.
> > @anushbhatia your further opinions?
> > # Additionally:
> > 
> > * Create a tag.MD file on the root level of the repo.
> > * Give the links to the most common tags including a side demo (similar to what we do for GitHub badges)
> >   React, HTML, CSS, JS, Python, TensorFlow, Keras, PyTorch, PyCaret, Scikit-Learn, opencv, socket.io
> > * Give a standalone PR for this, once this PR is merged
> 
> image
> 
> image
> 
> @smaranjitghose how about this?

Perfect!

**Comment by smaranjitghose** [comment]
> > > Nice work @Chaitanya31612!
> > > How do you propose we have multiple tags for the same card? Including the separate tag images themselves.
> > > @anushbhatia your further opinions?
> > > # Additionally:
> > > 
> > > * Create a tag.MD file on the root level of the repo.
> > > * Give the links to the most common tags including a side demo (similar to what we do for GitHub badges)
> > >   React, HTML, CSS, JS, Python, TensorFlow, Keras, PyTorch, PyCaret, Scikit-Learn, opencv, socket.io
> > > * Give a standalone PR for this, once this PR is merged
> > 
> > 
> > image
> > image
> > @smaranjitghose how about this?
> 
> @smaranjitghose can you please review this pull request and please explain what side demo are you talking about?

Ignore that. That comment was not meant for this PR😅

**Comment by smaranjitghose** [comment]
@Chaitanya31612 We need badges for:

Stacks/Libraries:
1. TensorFlow
2. PyTorch
3. OpenCV
4. React
5. Express
6. SQL
7. HTML
8. CSS
9. JS
10. Python
11. Julia
12. Qiskit
13. Streamlit
14. huggingface
15. GCP
16. AWS
17. Django
18. Flask
19. Fastapi

Topics: 
- Machine Learning
- Computer Vision
- Natural Language Processing
- Web  App
- Chatbot

If you wish to continue, you can either give a single PR to incorporate all the above.

**Comment by Chaitanya31612** [comment]
> @Chaitanya31612 We need badges for:
> 
> Stacks/Libraries:
> 
> 1. TensorFlow
> 2. PyTorch
> 3. OpenCV
> 4. React
> 5. Express
> 6. SQL
> 7. HTML
> 8. CSS
> 9. JS
> 10. Python
> 11. Julia
> 12. Qiskit
> 13. Streamlit
> 14. huggingface
> 15. GCP
> 16. AWS
> 17. Django
> 18. Flask
> 19. Fastapi
> 
> Topics:
> 
> * Machine Learning
> * Computer Vision
> * Natural Language Processing
> * Web  App
> * Chatbot
> 
> If you wish to continue, you can either give a single PR to incorporate all the above.

What do badges suggest, icon/logo of these stacks or tags for the project cards?

**Comment by smaranjitghose** [comment]
> > @Chaitanya31612 We need badges for:
> > Stacks/Libraries:
> > 
> > 1. TensorFlow
> > 2. PyTorch
> > 3. OpenCV
> > 4. React
> > 5. Express
> > 6. SQL
> > 7. HTML
> > 8. CSS
> > 9. JS
> > 10. Python
> > 11. Julia
> > 12. Qiskit
> > 13. Streamlit
> > 14. huggingface
> > 15. GCP
> > 16. AWS
> > 17. Django
> > 18. Flask
> > 19. Fastapi
> > 
> > Topics:
> > 
> > * Machine Learning
> > * Computer Vision
> > * Natural Language Processing
> > * Web  App
> > * Chatbot
> > 
> > If you wish to continue, you can either give a single PR to incorporate all the above.
> 
> What do badges suggest, icon/logo of these stacks or tags for the project cards?

Logo
